src/Swatch.tsx

```diff
@@ -153,17 +153,19 @@ function Swatch(
 ) {
   const clusterConfig = clusterConfiguration[stitchPattern];
   const clustered = !!clusterConfig.stitchCount;
-  const staggered = staggerLengths && staggerType != 'colorStretched'
```

Comment:
This line is the line that worries me in terms of non interwoven stitches. Moves even more logic onto CSS here? it possibly increases the coupling between the two

Reply:
though it might be better I don't have time to think about it